### DISCUSSION TITLE
Is there a better way to handle native chain tokens?

### DISCUSSION THREAD

**Discussion by helpfulBro** [aave/interface#659] [Ideas]
Regarding,
https://github.com/aave/interface/issues/654

It seems confusing to me that we have both native and wrapped, is there a better way to handle this?

Perhaps something like this?
![image](https://user-images.githubusercontent.com/25915621/162937836-7cc5e4ed-b5d5-4921-8709-9eab383db55f.png)

And in the supply/withdrawal modal the user can select to deposit NATIVE or WNATIVE?
![image](https://user-images.githubusercontent.com/25915621/162938120-b45ad38b-5072-42bd-a761-842ba683ad56.png)

**Comment by sendra** [comment]
In withdraw and repay and in borrow we have the toggle / option to do it with native or wrapped token.

The problem / confusion I think resides in that on the supply button on the user reserves section, it only allows to supply wrapped. As for supply we decided to separate wrapped and unwrapped, so users can see both balances.

**Reply by helpfulBro** [reply]
Which is not intuitive. 

If I originally supplied unwrapped then see it in my user reserves section, then tap supply I would expect to be able to supply the same version as I previously did.

Including a toggle in this modal would allow the user to complete the action without having to find the correct token in the list.
